src/_component/config/reusable.scss

```diff
@@ -0,0 +1,9 @@
+%rounder-border-list {
+    border-radius: 6px;
+    border:2px solid #707070;
```

Comment:
Line 3 and line 8 have a missing space between ":" and "2px", it's all about consistency.

Reply:
Your right! Thank you for your review!